src/main/java/com/dev/cinema/controllers/OrderController.java

```diff
@@ -0,0 +1,48 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.controllers.mappers.OrderMapper;
+import com.dev.cinema.model.ShoppingCart;
+import com.dev.cinema.model.dto.OrderResponseDto;
+import com.dev.cinema.service.OrderService;
+import com.dev.cinema.service.ShoppingCartService;
+import com.dev.cinema.service.UserService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final UserService userService;
+    private final ShoppingCartService shoppingCartService;
+    private final OrderMapper orderMapper;
+
+    @Autowired
```

Comment:
sometimes you use annotation with the constructor, sometimes not
let's be consistent